roadmap/implementers-guide/src/node/disputes/dispute-participation.md

```diff
@@ -35,7 +43,7 @@ Conclude.
 * If the result is `Unavailable`, return.
 * If the result is `Invalid`, [cast invalid votes](#cast-votes) and return.
 * Fetch the block number of `candidate_receipt.descriptor.relay_parent` using a [`ChainApiMessage::BlockNumber`][ChainApiMessage].
```

Comment:
doesn't seem to be used

Reply:
Indeed this was only needed for the `HistoricalValidationCode` API. Removed now.